cmd/service.go

```diff
@@ -501,6 +514,7 @@ func runServiceStopCmd(cmd *cobra.Command, args []string) {
 		}
 
 		status, err = db_local.StopServices(ctx, false, constants.InvokerService)
+		utils.FailOnErrorWithMessage(err, "could not stop db_local")
```

Comment:
change this whole block to 
```
	force := cmdconfig.Viper().GetBool(constants.ArgForce)
	if force {
		dashboardStopError := dashboardserver.StopDashboardService(ctx)
		status, dbStopError = db_local.StopServices(ctx, force, constants.InvokerService)
		dbStopError = utils.CombineErrors(dbStopError, dashboardStopError)
		utils.FailOnError(dbStopError)
	} else {
		dbState, dbStopError = db_local.GetState()
		utils.FailOnErrorWithMessage(dbStopError, "could not stop Steampipe service")

		dashboardState, err := dashboardserver.GetDashboardServiceState()
		utils.FailOnErrorWithMessage(err, "could not stop Steampipe service")

		if dbState == nil {
			fmt.Println("Steampipe service is not running.")
			return
		}
		if dbState.Invoker != constants.InvokerService {
			printRunningImplicit(dbState.Invoker)
			return
		}

		if dashboardState != nil {
			err = dashboardserver.StopDashboardService(ctx)
			utils.FailOnErrorWithMessage(err, "could not stop dashboard server")
		}

		var connectedClientCount int
		// check if there are any connected clients to the service
		connectedClientCount, err = db_local.GetCountOfThirdPartyClients(cmd.Context())
		utils.FailOnErrorWithMessage(err, "error during service stop")

		if connectedClientCount > 0 {
			printClientsConnected()
			return
		}

		status, err = db_local.StopServices(ctx, false, constants.InvokerService)
		utils.FailOnErrorWithMessage(err, "could not stop db_local")
	}

	switch status {
```

Reply:
i.e. remove
```
if dbStopError != nil {
		utils.ShowError(ctx, dbStopError)
		return
	}
``` 
from line 506 and add `utils.FailOnError(dbStopError)` to 472

Reply:
updated